db/seeds.rb

```diff
@@ -11,5 +11,5 @@
 end
 
 1000.times do
-  Gif.create!(url: Giphy.trending.url, user_id: rand(1)
+  Gif.create!(url: Giphy.random('technology').image_url.to_s, user_id: rand(1..99))
```

Comment:
Oh, nobody merge this pull request until refactor. It's 83 chars long
On Apr 14, 2016 10:26 PM, "Hound" <notifications@github.com> wrote:

> In db/seeds.rb
> <https://github.com/tiy-indy-bee-ror-feb16/Practicalgifs/pull/7#discussion_r59819919>
> :
>
> > @@ -11,5 +11,5 @@
> >  end
> >
> >  1000.times do
> > -  Gif.create!(url: Giphy.trending.url, user_id: rand(1)
> > +  Gif.create!(url: Giphy.random('technology').image_url.to_s, user_id: rand(1..99))
>
> Line is too long. [83/80]
> Prefer double-quoted strings unless you need single quotes to avoid extra
> backslashes for escaping.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tiy-indy-bee-ror-feb16/Practicalgifs/pull/7/files/c1ffbe0a5a61fe7ce8cba441d06fe8a242c781bd#r59819919>
>